src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1670,38 +1662,108 @@ def _test_single_bead_input(self, row: pd.Series) -> tuple[bool, str]:
+            string, Error message if the input is invalid.
         """
+
         try:
-            test_machine = AdditiveMachine(
+            allowed_types = [
+                SimulationType.SINGLE_BEAD,
+                SimulationType.POROSITY,
+                SimulationType.MICROSTRUCTURE,
+            ]
+            if row[ColumnNames.TYPE] in allowed_types:
+                if row[ColumnNames.TYPE] == SimulationType.SINGLE_BEAD:
+                    if np.isnan(row[ColumnNames.START_ANGLE]):
+                        row[ColumnNames.START_ANGLE] = MachineConstants.DEFAULT_STARTING_LAYER_ANGLE
+                    if np.isnan(row[ColumnNames.ROTATION_ANGLE]):
+                        row[
+                            ColumnNames.ROTATION_ANGLE
+                        ] = MachineConstants.DEFAULT_LAYER_ROTATION_ANGLE
+                    if np.isnan(row[ColumnNames.HATCH_SPACING]):
+                        row[ColumnNames.HATCH_SPACING] = MachineConstants.DEFAULT_HATCH_SPACING
+                    if np.isnan(row[ColumnNames.STRIPE_WIDTH]):
+                        row[
+                            ColumnNames.STRIPE_WIDTH
+                        ] = MachineConstants.DEFAULT_SLICING_STRIPE_WIDTH
+            else:
+                return (False, f"Invalid simulation type: {row[ColumnNames.TYPE]}.")
+
+            machine = AdditiveMachine(
                 laser_power=row[ColumnNames.LASER_POWER],
                 scan_speed=row[ColumnNames.SCAN_SPEED],
                 heater_temperature=row[ColumnNames.HEATER_TEMPERATURE],
                 layer_thickness=row[ColumnNames.LAYER_THICKNESS],
                 beam_diameter=row[ColumnNames.BEAM_DIAMETER],
+                starting_layer_angle=row[ColumnNames.START_ANGLE],
+                layer_rotation_angle=row[ColumnNames.ROTATION_ANGLE],
+                hatch_spacing=row[ColumnNames.HATCH_SPACING],
+                slicing_stripe_width=row[ColumnNames.STRIPE_WIDTH],
             )
 
-            test_material = AdditiveMaterial(
-                name=str(row[ColumnNames.MATERIAL]),
-            )
+            material = AdditiveMaterial(name=str(row[ColumnNames.MATERIAL]))
+
+            if row[ColumnNames.TYPE] == SimulationType.SINGLE_BEAD:
+                valid, error = self._validate_single_bead_input(machine, material, row)
+            if row[ColumnNames.TYPE] == SimulationType.POROSITY:
+                valid, error = self._validate_porosity_input(machine, material, row)
+            if row[ColumnNames.TYPE] == SimulationType.MICROSTRUCTURE:
+                valid, error = self._validate_microstructure_input(machine, material, row)
+            if not valid:
+                return (False, error)
+            return (True, "")
+
+        except ValueError as e:
+            return (False, (f"Invalid parameter combination: {e}"))
+
+    def _validate_single_bead_input(
+        self, machine: AdditiveMachine, material: AdditiveMaterial, row: pd.Series
```

Comment:
```suggestion
        self, machine: AdditiveMachine, material: AdditiveMaterial, input: pd.Series
```
Change `row` to `input` for the other `_validate` functions as well.